src/helpers/QueryHelper.ts

```diff
@@ -349,4 +349,8 @@ export default class QueryHelper {
 
         return inFilterExpression;
     };
+
+    static isNullorEmptyFilterExpression = (field: string) => {
+        return this.orFilterExpression([this.equalsFilterExpression(this.column(field), null), this.equalsFilterExpression(this.column(field), '')]);

```

Comment:
Místo `this.` u statických pls dej `QueryHelper.`
Ono je to bezpečnější v případě předávání pointerů na funkce. U jiných už to tam takto je.
3x na téhle řádce.

Reply:
Dobre, this by ale melo byt v pohode pokud jsou ty metody static coz jsou v kazdem pripade je to zmenene.